sensing/preprocessor/gnss/gnss_poser/include/gnss_poser/convert.hpp

```diff
@@ -13,17 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#pragma once
+#ifndef GNSS_POSER_CONVERT_HPP_
+#define GNSS_POSER_CONVERT_HPP_
 
 #include <GeographicLib/Geoid.hpp>
 #include <GeographicLib/MGRS.hpp>
 #include <GeographicLib/UTMUPS.hpp>
 
-#include "gnss_poser/gnss_stat.h"
-
 #include <geo_pos_conv/geo_pos_conv.hpp>
+#include <gnss_poser/gnss_stat.hpp>
+
+#include <rclcpp/logging.hpp>
+#include <sensor_msgs/msg/nav_sat_fix.hpp>
 
-namespace GNSSPoser
+namespace
```

Comment:
Rename was needed because of an error with multiple definitions, I certainly agree its not good to have anonymous namespaces in a header. I wasn't sure how much refactoring was needed for this porting PR. But from my understanding this header file seems to provide wrapper methods for accessing and manipulating data related to the geographiclib library. Not sure you think it would be better to make these methods inline for this porting PR?